packages/core/test/hotkeys/keyComboTagTests.tsx

```diff
@@ -18,11 +18,24 @@ import { render, screen } from "@testing-library/react";
 import { expect } from "chai";
 import * as React from "react";
 
-import { KeyComboTag } from "../../src/components/hotkeys";
+import { KeyComboTagInternal } from "../../src/components/hotkeys/keyComboTag";
 
 describe("KeyCombo", () => {
     it("renders key combo", () => {
-        render(<KeyComboTag combo="cmd+C" />);
+        render(<KeyComboTagInternal combo="cmd+C" platformOverride="Mac" />);
```

Comment:
Agreed its ugly. However, I took a look around and wasn't able to find a good way to mock `isMac`, but let me know if I missed something here. I did make the argument required on these functions so that we wouldn't forget to pass it and miss a spot.